zanata-adapter-properties/src/main/java/org/zanata/adapter/properties/PropWriter.java

```diff
@@ -43,6 +43,30 @@ public static void writeUTF8(final Resource doc, final File baseDir)
         write(doc, baseDir, true);
     }
 
+    /**
+     * Writes to given properties file of the given TranslationsResource
```

Comment:
Honestly, I'm not sure the about the default encoding. Didn't mentioned at all in org.fedorahosted.openprops.Properties